src/components/PatronBlock/PatronBlockModal.js

```diff
@@ -62,6 +79,10 @@ PatronBlockModal.propTypes = {
   onClose: PropTypes.func,
   patronBlocks: PropTypes.arrayOf(PropTypes.object),
   viewUserPath: PropTypes.func,
+  openOverrideModal: PropTypes.func,
```

Comment:
Could you please double check how i understood openOverrideModal and stripes need to be Required?

Reply:
Correct, thanks, fixed